### PR TITLE
[DOCS] Update How to install Great Expectations locally

### PR DESCRIPTION
**Pull request by NathanFarmer**
Changes proposed in this pull request:
- GREAT-340/GREAT-402
- Subtask: GREAT-411

### Definition of Done
- [X] My code follows the Great Expectations [style guide](https://docs.greatexpectations.io/en/latest/contributing/style_guide.html?highlight=style%20guide)
- [X] I have performed a [self-review](https://docs.greatexpectations.io/en/latest/contributing/contribution_checklist.html?highlight=checklist) of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] I have added [unit tests](https://docs.greatexpectations.io/en/latest/contributing/testing.html#contributing-testing-writing-unit-tests) where applicable and made sure that new and existing tests are passing.
- [X] I have run any local integration tests and made sure that nothing is broken.

### CODE REVIEW

File: docs/guides/setup/installation/local.md [hunk 1]
```diff
@@ -6,41 +6,97 @@ import Congratulations from '/docs/guides/setup/components/install_congrats.md'
...
-# or if multiple versions of Python installed
+or
+
+```console
+python3 -m ensurepip --upgrade
 python3 -m pip install great_expectations
 ```
 
+</TabItem>
+<TabItem value="conda">
```
Comment (donaldheppner) [inline docs/guides/setup/installation/local.md:90]: ❤️

File: docs/guides/setup/installation/local.md [hunk 2]
```diff
@@ -6,41 +6,97 @@ import Congratulations from '/docs/guides/setup/components/install_congrats.md'
...
+  ]}>
+<TabItem value="pip">
+
+Once you have confirmed that Python 3 is installed locally, you will run two commands that will:
+
+1. Ensure that pip is installed and upgraded to the latest version
+2. Install Great Expectations using pip
+
+Depending on whether you found that you needed to run `python` or `python3` in the previous step, you will run either:
+
```
Comment (donaldheppner) [inline docs/guides/setup/installation/local.md:58]: Do you think we should advise to install ge into a virtualenv for pip, to be consistent with the conda instructions?

**Review by donaldheppner — CHANGES_REQUESTED** [review]

**Review by anthonyburdi — COMMENTED** [review]
LGTM pending resolution of Don's comment.

**Review by alexsherstinsky — APPROVED** [review]
LGTM

**Review by anthonyburdi — APPROVED** [review]
LGTM!

File: docs/guides/setup/installation/local.md (hunk 2 above)
Comment (NathanFarmer) [reply docs/guides/setup/installation/local.md:58]: I have added instructions for installation into a virtual environment using `venv`.

**Review by donaldheppner — APPROVED** [review]

**Pull request merged**